functional_widget/pubspec.yaml

```diff
@@ -2,26 +2,27 @@ name: functional_widget
 description: A code generator that generates widget classes from their implementation as a function.
 author: Remi Rousselet <darky12s@gmail.com>
 homepage: https://github.com/rrousselGit/functional_widget/tree/master/functional_widget
-version: 0.5.0
+version: 0.5.1
 
 environment:
     sdk: '>=2.1.0 <3.0.0'
 
 dependencies:
-    functional_widget_annotation: ^0.5.0
-    source_gen: ^0.9.0
-    build_config: '>=0.3.0 <1.0.0'
-    code_builder: '>=3.1.3 <4.0.0'
-    build: '>=0.12.6 <2.0.0'
-    analyzer: '>=0.32.2 <0.34.0'
+    functional_widget_annotation:
```

Comment:
we cannot publish to pub with a path version
Using version number is required

Reply:
okay!

Reply:
I'll put it back in (I thought it could have been related to one of the problems with the versioning and then thought maybe it worked ok like that)